src/scenarios/emqttb_scenario_persistent_session.erl

```diff
@@ -0,0 +1,279 @@
+
+%%================================================================================
+%% Type declarations
+%%================================================================================
+
+-define(PUB_GROUP, pers_sess_pub_group_pub).
+-define(SUB_GROUP, pers_sess_pub_group_sub).
+
+-define(PUB_THROUGHPUT, emqttb_pers_sess_pub_throughput).
+-define(SUB_THROUGHPUT, emqttb_pers_sess_sub_throughput).
+
+-record(s,
+        { produced = 0
+        , consumed = 0
+        , to_consume = 0
+        }).
+
+%%================================================================================
+%% behavior callbacks
+%%================================================================================
+
+name() ->
+  persistent_session.
+
+model() ->
+  #{ pub =>
+       #{ qos =>
+            {[value, cli_param],
+             #{ oneliner => "QoS of the published messages"
+              , type => emqttb:qos()
+              , default => 1
+              , cli_operand => "pub-qos"
+              }}
+        , msg_size =>
+            {[value, cli_param],
+             #{ oneliner => "Size of the published message in bytes"
+              , type => non_neg_integer()
+              , cli_operand => "size"
+              , cli_short => $s
+              , default => 256
+              }}
+        , pubinterval =>
+            {[value, cli_param],
+             #{ oneliner => "Message publishing interval"
+              , type => emqttb:interval()
+              , default_ref => [interval]
+              , cli_operand => "pubinterval"
+              , cli_short => $i
+              }}
+        , n =>
+            {[value, cli_param],
+             #{ oneliner => "Number of publishers"
+              , type => emqttb:n_clients()
+              , default => 10
+              , cli_operand => "num-publishers"
+              , cli_short => $P
+              }}
+        , set_pub_latency =>
+            {[value, cli_param],
+             #{ oneliner => "Try to keep publishing time at this value (ms)"
+              , type => integer()
+              , default => 100
+              , cli_operand => "publatency"
+              }}
+        , pub_autorate =>
+            {[value, cli_param, pointer],
+             #{ oneliner    => "ID of the autorate config used to tune publish interval"
+              , type        => atom()
+              , default     => default
+              , cli_operand => "pubautorate"
+              , target_node => [autorate]
+              }}
+        , topic_suffix =>
+            {[value, cli_param],
+             #{ oneliner => "Suffix of the topic to publish to"
+              , type => binary()
+              , default => <<"%h/%n">>
+              , cli_operand => "topic"
+              , cli_short => $t
+              }}
+        , pub_time =>
+            {[value, cli_param],
+             #{ oneliner => "Suffix of the topic to publish to"
+              , type => non_neg_integer()
+              , default => 5_0000
```

Comment:
```suggestion
              , default => 50_000
```

Reply:
I found that shorter bursts of traffic lead to more stable tests, when too many messages are queued up, EMQX tends to start dropping messages.

Reply:
I found that shorter bursts of traffic lead to more stable tests, when too many messages are queued up, EMQX tends to start dropping messages.